Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -105,21 +105,37 @@ open class PassiveLocationManager: NSObject {
     
     var lastRawLocation: CLLocation?
     
+    /**
+     A closure to be called when manual location was successfully set.
+     
+     - parameter success: Boolean value, which is set to `true` in case if manual location was
+     successfully set, otherwise `false`.
+     */
+    public typealias UpdateLocationCompletionHandler = (_ success: Bool) -> Void
```

Comment:
Would it be useful to have this completion handler take a `Result<Error, CLLocation>` instead of a Boolean?

Reply:
Please check my reply [here](https://github.com/mapbox/mapbox-navigation-ios/pull/3342/files#r710530286). I like your proposal, but this completion handler is only used in one method, which uses only one location instead of their collection, so it'll provide excessive information, as user already has similar value as function parameter.